Kickstarter-iOS/Views/Controllers/BackerDashboardProjectsViewController.swift

```diff
@@ -38,7 +38,15 @@ internal final class BackerDashboardProjectsViewController: UITableViewControlle
   internal override func bindViewModel() {
     super.bindViewModel()
 
-    self.refreshControl?.rac.refreshing = self.viewModel.outputs.isRefreshing
+    self.viewModel.outputs.isRefreshing
+      .observeForUI()
+      .observeValues { [weak self] isRefreshing in
+        if isRefreshing {
+          self?.refreshControl?.beginRefreshing()
+        } else {
+          self?.refreshControl?.endRefreshing()
+        }
+    }
```

Comment:
the output didn't trigger the rac extension when it was written that way. I'm not sure why.